contracts/libs/arrays/ArrayHelper.sol

```diff
@@ -103,6 +103,14 @@ library ArrayHelper {
         array_[0] = elem_;
     }
```

Comment:
Should these 3 functions be in a TypeCaster?

Reply:
done